test/test.js

```diff
@@ -121,6 +121,19 @@ describe('Puppeteer', function() {
       expect(fs.readdirSync(userDataDir).length).toBeGreaterThan(0);
       rm(userDataDir);
     }));
+
+    // Disabled on Windows until we find a good use case on this platform. The
+    // TZ environment variable is currently ignored, see
+    // https://bugs.chromium.org/p/chromium/issues/detail?id=125614
+    (process.platform === 'win32' ? xit : it)('env option', SX(async function() {
```

Comment:
Lets not add this test - it adds more debt.
The patch is small and mostly does plumbing, so we can tolerate landing this without a test.

Reply:
Alright, test removed! (commit amended)